.github/actions/merge-base-branch/action.yml

```diff
@@ -0,0 +1,26 @@
+name: 'Merge from Base Branch if Action Running on a PR'
+description: 'Merge from the base branch the PR is being merged into.'
+runs:
+  using: 'composite'
+  steps:
+    - id: merge
+      name: Merge
+      run: |
+        if [[ ${{ github.event_name}} != "pull_request" ]]
+        then
+          echo "Skipping merge because not triggered by a PR event."
+          exit 0
+        fi
+        echo "Base Repository: ${{ github.repository }}"
```

Comment:
Just to confirm we are expecting Base repo to be `oppia/oppia`, right?

Reply:
It usually will be, but sometimes contributors (like me!) open PRs against their own forks for debugging since those tests bypass the oppia/oppia queue